allauthdemo/templates/account/email.html

```diff
@@ -0,0 +1,115 @@
+{% extends "django_classified/_base.html" %}
+
+{% load i18n bootstrap4 %}
+
+{% block head_links %} 
+
+{% endblock %}
+
+{% block title %}{% trans "Account" %}{% endblock title %}
+
+
+{% block body %}
+
+
+	{% block content %}
+	
+	 {% block messages %}
+		{% if messages %}
+			{% for message in messages %}
+			<div class="alert alert-warning alert-dismissible fade show" role="alert">
+					  <strong>{{message}}</strong>
+					<button type="button" class="close" data-dismiss="alert" aria-label="Close">
+						<span aria-hidden="true">&times;</span>
+					</button>
+			</div>
+			{% endfor %}
+		{% endif %}
+	  {% endblock %}
+	
+	<h3>{% trans "E-mail addresses" %}</h3>
+	
+	{% if user.emailaddress_set.all %}
+	
+	<p>{% trans 'The following e-mail addresses are associated with your account:' %}</p>
+	
+	<form class="form-horizontal" action="{% url 'account_email' %}" method="post">
+	{% csrf_token %}
+	
+	  {% for emailaddress in user.emailaddress_set.all %}
+	  <div class="ctrlHolder">
+	  
+	  <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
+	
+		{{ emailaddress.email }}
+	
+		{% if emailaddress.verified %}
+		  <span class="verified">&nbsp;<i class="fa fa-check" aria-hidden="true"> (Verified) &nbsp;</i></span>
+		{% else %}
+		  <span class="unverified">&nbsp;<i class="fa fa-circle-o-notch" aria-hidden="true"> (Not verified!) &nbsp;</i></span>
+		{% endif %}
+		
+		<input id="email_radio_{{forloop.counter}}" type="radio" name="email" {% if emailaddress.primary or user.emailaddress_set.count == 1 %}checked="checked"{%endif %} value="{{emailaddress.email}}"/>
+	
+		{% if emailaddress.primary %}
+		  <span class="primary">&nbsp;Primary e-mail</span>
```

Comment:
I think it should be translatable.

Reply:
Done, checked for other trans absences on template, none found.